.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Should we build the plugin for the same platforms as Trivy to maintain backwards compatibility?

Reply:
I think we can add cross platform support later on if asked for. Feels like feature creep for this PR.